core/src/io/anuke/mindustry/world/blocks/distribution/ItemBridge.java

```diff
@@ -236,6 +236,10 @@ public void drawLayer(Tile tile){
         Draw.rect(endRegion, tile.drawx(), tile.drawy(), i * 90 + 90);
         Draw.rect(endRegion, other.drawx(), other.drawy(), i * 90 + 270);
 
+        int opacityPercentage = Core.settings.getInt("bridgeopacity");
+        if(opacityPercentage == 0) return;
+
+        Draw.alpha(opacityPercentage / 100f);
```

Comment:
This is incorrect. Alpha is being set a few lines above this call. Instead of setting the alpha here, you should put the opacity check up there and multiply the base opacity by the preferred settings opacity.

Reply:
Isn't that for the arrows?

Reply:
No, it's for the entire bridge.

Reply:
Hmm